app/assets/stylesheets/_main.scss

```
@@ -325,6 +325,10 @@ html * {
   display: flex;
 }
 
+.justify-content-space-between {
+  justify-content: space-between;
+}
+
```

Just a nitpick really but I don't much like the style of using many CSS utility classes. Better to do something like

```suggestion
.press-notice-status {
  display: flex;
  justify-content: space-between;
}
```

and then use the more specific class name rather than having to do `class="display-flex justify-content-space-between"` everywhere.

---

I think this would be a good topic for debate! 
I personally prefer the descriptive and reusable nature of these css classes that are agnostic to the component but as you say some people prefer the other way you suggested.
Using `.press-notice-status` might not be appropriate for another element on another page that requires these css classes. Maybe if we had a more generic name i.e. `status-panel` then we can reuse without being too specific.

Happy to discuss as there's never just the one best answer for these type of things

---

> Maybe if we had a more generic name 

Yeah definitely — I haven't looked through everything to see if there is an appropriate generic component (and I acknowledge that's part of the problem). We do have other classes that use the same combination of attributes, but it's not immediately clear whether it makes sense for them to be shared in these specific cases.

I think at the moment we have a mixture of both ways of doing things, which isn't ideal either. Worth thinking about at some point — maybe something to discuss at dev club.

---

Yep agreed.
You're probably not a fan of tailwind css then 😆

---

I'm not, ideally — but I do think that if we want to primarily use utility classes then we should consider it. It might reduce the hassle of creating the classes, at least.